clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/deploy/asg/AsgConfigHelper.java

```diff
@@ -332,6 +332,35 @@ private static String createDefaultSuffix() {
         .collect(Collectors.toList());
   }
 
+  protected static List<AmazonBlockDevice> convertBlockDevices(
```

Comment:
Javadoc would be helpful here (we have it for the other variant).

Reply:
Added Javadoc